packages/mobile/ios/Podfile

```diff
@@ -22,7 +22,8 @@ target "celo" do
   use_frameworks!
 
   # Headers only version of OpenSSL-Universal to workaround libcrypto / libssl conflict between Flipper and react-native-fast-crypto
-  pod 'OpenSSL-Universal', :git => "https://github.com/celo-org/OpenSSL-headers.git"
+  # pod 'OpenSSL-Universal', :git => "https://github.com/celo-org/OpenSSL-headers.git"
```

Comment:
I am not sure why we were using our fork but we should check. cc @jeanregisser

Reply:
From @jeanregisser 
> OpenSSL-Universal was only needed to make Flipper work
however react-native-fast-crypto already includes libssl and libcrypto
so we would get linking errors because of this

No linking errors were observed.